Dockerfile

```diff
@@ -24,5 +24,14 @@ WORKDIR /go/src/github.com/insolar/insolar
 RUN  set -eux; \
      groupadd -r insolar --gid=999; \
      useradd -r -g insolar --uid=999 --shell=/bin/bash insolar
-COPY --from=build /go/src/github.com/insolar/insolar/bin/insolar /go/src/github.com/insolar/insolar/bin/insolard /go/src/github.com/insolar/insolar/bin/keeperd /go/src/github.com/insolar/insolar/bin/pulsard /usr/local/bin/
+COPY --from=build /go/src/github.com/insolar/insolar/application/api/spec/api-exported.yaml /app/api-exported.yaml
```

Comment:
I would prefer to have it mounted as a volume from ConfigMap and not to be placed here at build stage. What do you think?

Reply:
it's just binary asset which required and no reason to not put it outside image, probably best option to embed it inside binary (jira task?)

Reply:
yeah, sure